core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -45,6 +45,13 @@
 -export([delete_system_media_references/0]).
 -export([migrate_system/0]).
 
+-export([binding/1]).
+
+binding(migrate) -> <<"maintenance.migrate">>;
```

Comment:
I feel like these should be database names. Like, for webhooks, the webhooks application should call `refresh_binding(?KZ_WEBHOOKS_DB)` when binding, which would bind to `<<"maintenance.refresh.", Db/binary>>` here.